src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -471,8 +476,13 @@ class Resize(tvt_v2.Transform, NumpytoTVTensorMixin):
     TODO : optimize logic to torcivision pipeline
 
     Args:
-        scale (int or tuple): Images scales for resizing with (height, width). Defaults to None
-        scale_factor (float or tuple[float]): Scale factors for resizing with (height, width).
+        input_size (int or Sequence[int]): Final transformed image size.
+            It can be set in each recipe or cli, and is assigned in otx.core.data.factory.TransformLibFactory.
+        resize_size (int or Sequence[int]): Images scales for resizing with (height, width).
+            Defaults to None.
+        resize_size_scale (Sequence[float]): Scale factors to adjust `resize_size` using `input_size`
+            when `resize_size` is None. Defaults to (1.0, 1.0).
+        scale_factor (float or Sequence[float]): Scale factors for resizing with (height, width).
```

Comment:
The names and descriptions of the arguments here don't really make sense to me at first glance (I don't understand them), and I think there needs to be more clarification on what combinations work, what doesn't, and what combinations are recommended.

Reply:
There are too many parameters for resizing, "resize size", "input_size", "resize size scale" ... It became complicated. Why do we need so many parameters for just resizing image?